src/server/profiles/account/account.config.js

```diff
@@ -0,0 +1,89 @@
+const { route_args, common_args, write_args } = require('../common.arguments');
+const { read_scopes, write_scopes } = require('../common.scopes');
+const { CONFIG_KEYS, VERSIONS } = require('../../../constants');
+const resource_args = require('./account.arguments');
+const controller = require('./account.controller');
+
+let write_only_scopes = write_scopes('Account');
+let read_only_scopes = read_scopes('Account');
+
+let commonArgsArray = Object.getOwnPropertyNames(common_args)
```

Comment:
these should be snake case to follow convention

Reply:
Ok. Will do snake case.